.circleci/config.yml

```diff
@@ -4,11 +4,11 @@ executors:
   basic-executor:
     resource_class: small
     docker:
-      - image: cimg/ruby:3.2.2-browsers
+      - image: cimg/ruby:3.3.1-browsers
   test-executor:
     resource_class: medium
     docker:
-      - image: checkclientqualifiesdocker/circleci-image:puppeteer-2271
+      - image: checkclientqualifiesdocker/circleci-image:EL-1495-ruby-331
```

Comment:
I am assuming this changes before we merge to main, is that correct?

Reply:
It's a chicken and egg problem, so no it can't sadly.